<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planWithFriendsDialog</class>
 <widget class="QDialog" name="planWithFriendsDialog">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>229</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>598</width>
    <height>229</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>598</width>
    <height>229</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plan with Friends</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/resources.qrc">
    <normaloff>:/planner.png</normaloff>:/planner.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="friendListWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="viewFriendPlanBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>View Friend's Plan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteFriendBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete Friend</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addFriendBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Friend</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
